src/main/java/org/sciborgs1155/robot/Ports.java

```diff
@@ -1,6 +1,33 @@
 package org.sciborgs1155.robot;
 
+import java.util.HashMap;
+import java.util.Map;
+
 public final class Ports {
+
+  public static Map<Integer, String> idToName;
+
+  static {
```

Comment:
Since there is limited space on the actual dashboards for viewing logs, I want you to cut down on the unnecessary details. You could probably remove the ending "motor" for each nickname b/c that's the only relevant thing we control about those subsystems.